### PR TITLE
DOCS/man/options.rst: add gpu-next differences to --scale-antiring

### PR DESCRIPTION
**Pull request by ghost**
Cherry-picked from https://github.com/mpv-player/mpv/pull/14863 and rephrased/expanded upon.

### CODE REVIEW

File: DOCS/man/options.rst [hunk 1]
```diff
@@ -5595,6 +5595,10 @@ them.
     Note that this doesn't affect the special filters ``bilinear`` and
     ``bicubic_fast``, nor does it affect any polar (EWA) scalers.
 
+    On ``--vo=gpu-next``, this affects polar (EWA) scalers. Certain filter
```
Comment (kasper93) [inline DOCS/man/options.rst:5598]:
```suggestion
    On ``--vo=gpu-next``, this also affects polar (EWA) scalers. Certain filter
```

And we could mention for ortho scalers, it is disabled for dscale.

**Review by kasper93 — APPROVED** [review]

**Comment by hooke007** [comment]
I am sure it is difficult to understand `separable (orthogonal) filters` for most users.

**Comment by kasper93** [comment]
Good point, @norinoriko do you want to resolve the above concern somehow?

**Comment by hooke007** [comment]
> Do most users know what a polar/2D filter is

Why not? `ewa` is mentioned.

**Pull request merged**